projects/keycloak/build.sh

```diff
@@ -18,21 +18,33 @@
 # Retrieve JDK-17
 wget https://download.java.net/openjdk/jdk17/ri/openjdk-17+35_linux-x64_bin.tar.gz
 tar -zxf openjdk-17+35_linux-x64_bin.tar.gz
-JAVA_HOME=$SRC/keycloak/jdk-17
+cp -r jdk-17 $OUT/
+JAVA_HOME=$OUT/jdk-17
+PATH=$JAVA_HOME/bin:$PATH
 
 # Build Keycloak
 MAVEN_ARGS="-Djavac.src.version=17 -Djavac.target.version=17 "
 MAVEN_ARGS=$MAVEN_ARGS"-DskipTests -Dgpg.skip -Dmaven.source.skip "
 MAVEN_ARGS=$MAVEN_ARGS"-DskipExamples -DskipTestsuite"
 $MVN clean package $MAVEN_ARGS
 
+# Dependency
+wget https://repo1.maven.org/maven2/org/bouncycastle/bc-fips/1.0.2.3/bc-fips-1.0.2.3.jar
```

Comment:
Please specify by way of commenting which fuzzers each of these were initially added for.

Reply:
Removed when build.sh is changed in #422.